en/news/_posts/2018-11-08-snap.md

```diff
@@ -9,7 +9,7 @@ lang: en
 
 We released the official snap package of Ruby language.
 
-https://snapcraft.io/ruby
+[https://snapcraft.io/ruby](https://snapcraft.io/ruby)
```

Comment:
This can be simplified to `<https://snapcraft.io/ruby>`.

Reply:
TIL, fixed.